README.md

```diff
@@ -1,3 +1,6 @@
+![Linux Post Commit Checks](https://github.com/intel/llvm/workflows/Linux%20Post%20Commit%20Checks/badge.svg)
+![Generate Doxygen documentation](https://github.com/intel/llvm/workflows/Generate%20Doxygen%20documentation/badge.svg)
+
```

Comment:
It probably makes sense to create some "Project Status" subbullet under oneAPI DPC++ project below and add badges there (line 10 in unmodified file).
Though currently we host only DPC++ project here, there could be more Intel LLVM-based projects added in the future.

Reply:
Here is how it looks like: https://github.com/intel/llvm/blob/07bca65467d5dce4836ae95415ffda32b5a7d366/README.md
I'm not sure sub-bullet is needed or adds any value.